corehq/apps/data_dictionary/models.py

```diff
@@ -125,6 +124,10 @@ def get_or_create(cls, name, case_type, domain):
                     name=name, case_type__name=case_type, case_type__domain=domain
                 )
             except CaseProperty.DoesNotExist:
+                from corehq.apps.hqcase.case_helper import CaseCopier
+                if name in [CaseCopier.COMMCARE_CASE_COPY_PROPERTY_NAME]:
```

Comment:
Nit
```suggestion
                if name == CaseCopier.COMMCARE_CASE_COPY_PROPERTY_NAME:
```

Reply:
As I was writing this code I thought of doing the same thing, but changed my mind to "keep it generic" as in "just add to this list if you want some additional restricted property restricted". Looking at it now I realise I would probably look at this code in the future and think "why did this person do it this way?"...and maybe that's a good indication to do it differently.